base/leaky_bucket.go

```diff
@@ -123,13 +121,6 @@ type LeakyBucketConfig struct {
 	DDocDeleteErrorCount int
 	DDocGetErrorCount    int
 
-	// Emulate TAP/DCP feed de-dupliation behavior, such that within a
-	// window of # of mutations or a timeout, mutations for a given document
-	// will be filtered such that only the _latest_ mutation will make it through.
-	TapFeedDeDuplication bool
-	TapFeedVbuckets      bool     // Emulate vbucket numbers on feed
-	TapFeedMissingDocs   []string // Emulate entry not appearing on tap feed
```

Comment:
TapFeedMissingDocs doesn't seem like tap vs. DCP functionality, it's more about tests having the ability to simulate a document not arriving over the feed.  I understand we don't have any real test usage of this today - are you thinking we'd just rebuild if/when it's needed?

Reply:
I was thinking we'd rebuild it when we needed it. I'm OK leaving this code in, and I think that it could be used to build artificial DCP feeds.